backend/src/main/kotlin/com/tngtech/apicenter/backend/config/RestResponseExceptionHandler.kt

```diff
@@ -6,21 +6,34 @@ import org.springframework.web.bind.annotation.ControllerAdvice
 import org.springframework.web.bind.annotation.ExceptionHandler
 import java.time.Instant
 import java.time.format.DateTimeFormatter
+import mu.KotlinLogging
+import java.util.*
+
+private val logger = KotlinLogging.logger {  }
+class SpecificationNotFoundException(val specificationId: UUID, val version: String? = "") : RuntimeException()
```

Comment:
These exceptions must be defined somewhere in the domain layer, as they are (obviously) domain exceptions.